src/main/python/rlbot/training/status_rendering.py

```diff
@@ -1,10 +1,22 @@
 from collections import namedtuple
+from contextlib import contextmanager
 from typing import List
 
 from rlbot.utils.rendering.rendering_manager import RenderingManager
 
 Row = namedtuple('Row', 'exercise_name status_str status_color_func')
 
+@contextmanager
+def training_status_renerer_context(exercise_names: List[str], renderman: RenderingManager):
```

Comment:
Little typo here, and the other places where it's used.

Reply:
Thanks for spotting that. Fixed.